src/components/Schedule/Schedule.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+
+import { Svg } from 'components/Svg';
+import { breadcrumbs } from 'redux/modules';
+import { useAppDispatch } from 'redux/store';
+import { useBaseBreadcrumbs } from 'hooks';
+
+export const Schedule = (): JSX.Element => {
+  const { id } = useParams();
+  const { t } = useTranslation('app');
+  const dispatch = useAppDispatch();
+  const baseBreadcrumbs = useBaseBreadcrumbs();
+
+  useEffect(() => {
+    if (id && baseBreadcrumbs && baseBreadcrumbs.length > 0) {
```

Comment:
you can use     if (id && baseBreadcrumbs?.length > 0) {

Reply:
done